x/name/module.go

```diff
@@ -152,10 +152,12 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 	simulation.RandomizedGenState(simState)
 }
 
-// ProposalContents returns all the name content functions used to
+// ProposalContents returns all the v1beta1 name content functions used to
 // simulate governance proposals.
-func (am AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent {
-	return simulation.ProposalContents(am.keeper)
+func (am AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalMsg {
+	// This is for stuff that uses the v1beta1 gov module's Content interface.
+	// This module use the v1 gov pattern and no longer needs this.
+	return []simtypes.WeightedProposalMsg{}
 }
```

Comment:
This `ProposalContents` method isn't needed anymore. We should probably delete them from all the modules that don't have any proposals. We can do the others in other PRs, but we should at least start with this one.

Also, this `ProposalContents` method is only used to satisfy the `HasProposalContents` interface. That expects the `ProposalContents ` method to return `[]simtypes.WeightedProposalContent` even though that type is deprecated. By making it return `[]simtypes.WeightedProposalMsg` instead, it makes this method completely ignored and unused. There's a new `HasProposalMsgs` interface, satisfied by a `ProposalMsgs` method that returns `[]simtypes.WeightedProposalMsg`. I don't know much about that method, but I expect we'll want to utilize it when get get around to trying to fix the sims. 

For now, though, we can get rid of any empty `ProposalContents` that we come across.